jobs/tests/test_utils.py

```diff
@@ -170,15 +170,15 @@
 
     # Thunderbird Mac OS X Release
     {
-        "id": "thunderbird_10-0-12esr_macosx_pt-br",
+        "id": "thunderbird_esr_10-0-12esr_macosx_pt-br",
         "source": {
             "product": "thunderbird",
         },
         "target": {
             "version": "10.0.12esr",
             "platform": "macosx",
             "locale": "pt-BR",
-            "channel": "release"
+            "channel": "esr"
```

Comment:
I'm pretty sure that .json file is a lie...we use "release" in Balrog. I'm going to double check an actual build to be sure.